### PR TITLE
[docs][platform] PR for Enable high availability #16133

### LINKED ISSUE #16133: [Docs] Enable high availability
### Description

In the page describing [YBA high availability](https://docs.yugabyte.com/preview/yugabyte-platform/manage-deployments/high-availability/), it mentions "xCluster replication". The HA feature does not use xCluster replication - xCluster is a YugabyteDB feature. This should probably say "asynchronous replication" instead.

### PR DESCRIPTION
**Pull request by ddhodge**
Fix #16133

@netlify /preview/yugabyte-platform/manage-deployments/high-availability/

### CODE REVIEW

**Comment by ddhodge** [comment]
@sb-yb Would the High Availability page more logically go under Administer section given it is about YBA admin, as opposed to managing universe deployment.

**Comment by sb-yb** [comment]
@ddhodge  Administer section makes more sense.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 1]
```diff
@@ -11,15 +11,14 @@ menu:
 type: docs
 ---
 
-Yugabyte’s distributed architecture enables database clusters (also referred to as universes) to have extremely high availability.
+YugabyteDB's distributed architecture enables database clusters (also referred to as universes) to have extremely high availability.
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:14]: This is unrelated to YBA HA and not sure we should even mention it here. WDYT? 
I know you are just editing existing paragraph.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 2]
```diff
@@ -29,20 +28,20 @@ When you promote a standby instance to active, YugabyteDB Anywhere restores your
 
 Before configuring a high-availability cluster, ensure that you have the following:
 
-* YugabyteDB Anywhere versioin 2.5.3.1 or later.
-* [Multiple YugabyteDB Anywhere instances](../../install-yugabyte-platform/) to be used in the high-availability cluster.
-* YugabyteDB Anywhere VMs can connect to each other over the port that YugabyteDB Anywhere is typically reachable (port 80 and 443, for example).
-* All YugabyteDB Anywhere instances are running the same version of YugabyteDB Anywhere software. Note that it is generally recommended to upgrade all YugabyteDB Anywhere instances in the high-availability cluster at approximately the same time.
+- YugabyteDB Anywhere version 2.5.3.1 or later.
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:31]: Do we need this line anymore given that 2.6 is also EOL?

Comment (sb-yb) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:31]: Basically all our active releases now support this feature. So why add useless info about when it was first introduced.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 3]
```diff
@@ -29,20 +28,20 @@ When you promote a standby instance to active, YugabyteDB Anywhere restores your
...
+- [Multiple YugabyteDB Anywhere instances](../../install-yugabyte-platform/) to be used in the high-availability cluster.
+- YugabyteDB Anywhere VMs can connect to each other over the port that YugabyteDB Anywhere is typically reachable (port 80 and 443, for example).
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:33]: YugabyteDB Anywhere UI is typically reachable

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md [hunk 4]
```diff
@@ -126,22 +124,22 @@ Even though you can perform an upgrade of all YugabyteDB Anywhere instances simu
 The following is the detailed upgrade procedure:
 
 1. Stop the high-availability synchronization. This ensures that only backups of the original YugabyteDB Anywhere version are synchronized to the standby instance.
-2. Upgrade the active instance. Expect a momentary lapse in availability for the duration of the upgrade. If the upgrade is successful, proceed to step 3. If the upgrade fails, perform the following:
+1. Upgrade the active instance. Expect a momentary lapse in availability for the duration of the upgrade. If the upgrade is successful, proceed to step 3. If the upgrade fails, perform the following:
```
Comment (sb-yb) [inline docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:127]: 2?

Comment (sb-yb) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:127]: Or it will auto number?

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:14]: I agree. Made an edit to the next paragraph as well.

File: docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md (hunk 4 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md:127]: Using 1. for all steps engages auto numbering, yes

**Review by aishwarya24 — APPROVED** [review]
